dev-utils/bitrot.js

```diff
@@ -205,7 +205,7 @@ function loadSupportedDoc () {
         results.push({ name: n, versions: row[1] })
       })
     } else {
-      match = /^https:\/\/.*\[(.*)\]$/.exec(row[0])
+      match = /^https:\/\/.*\[(.*)\]$/.exec(row[0].trim())
```

Comment:
Looks good. Out of curiosity: is there any reason we want to keep starting/trailing spaces in cell values?

Reply:
Nope, there is no reason to keep surrounding spaces in cell values.